dev-tools/common/test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package common
+
+import (
+	"context"
+	"fmt"
+
+	"io"
+	"os"
+	"os/exec"
+
+	"github.com/magefile/mage/mg"
+	"github.com/pkg/errors"
+)
+
+func GoUnitTest(ctx context.Context) error {
+	mg.Deps(InstallGoTestTools)
+
+	fmt.Println(">> go test:", "Unit Testing") //nolint:forbidigo // just for tests
```

Comment:
I think if you are using a Print* methods and a writer, it should not mark any usage.

```golang
out := os.stdout
fmt.Fprintln(out, >> go test:", "Unit Testing) 
```

The above allow you to enable the output or not.